data/src/main/java/co/nimblehq/compose/crypto/data/repository/CoinRepositoryImpl.kt

```diff
@@ -30,4 +32,20 @@ class CoinRepositoryImpl(
             page = page
         )
     }.map { it.toModels() }
+
+    override fun getTrends(
```

Comment:
@Wadeewee please add unit test, for getCoins API too 🙏 

sample here: https://github.com/nimblehq/android-templates/pull/291/files#diff-4c5e3948d43d0c3d46be7c596911cc18084a0281ccc7edccc19a1dfe6c49f0bd